content/getting-started/building-a-cordova-app.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Building a Cordova app 
+description: How to build a Cordova app with codemagic.yaml
+weight: 9
+---
+
+## Setting up a Cordova project
+
+The apps you have available on Codemagic are listed on the Applications page. See how to add additional apps [here](./adding-apps-from-custom-sources).
+
+1. On the [Applications page](https://codemagic.io/apps), click **Set up build** next to the app you want to start building. 
+2. On the popup, select **Cordova** as the project type and click **Continue**.
+3. Create a [`codemagic.yaml`](./yaml) file and add in it the commands to build, test and publish your project. See the full workflow examples below.
+4. Commit the configuration file to the root of your repository.
+5. Back in app settings in Codemagic, scan for the `codemagic.yaml` file by selecting a **branch** to scan and clicking the **Check for configuration file** button at the top of the page. Note that you can have different configuration files in different branches.
+6. If a `codemagic.yaml` file is found in that branch, you can click **Start your first build** and select the **branch** and **workflow** to build.
+7. Finally, click **Start new build** to build the app.
+
+{{<notebox>}}
+**Automatic build triggers**
+
+Note that you need to set up a [webhook](../building/webhooks) for automatic build triggers. Click the **Create webhook** button on the right sidebar in app settings to add a webhook (not available for apps added via SSH/HTTP/HTTPS).
+
+{{</notebox>}}
+
+## Code signing and publishing Cordova Android and iOS apps
+
+To build, code sign, and publish Cordova Android and iOS apps:
+
+* All Android apps need to be signed before release. See the [Android codesigning docs](../code-signing/android-code-signing/) or the sample workflow below for more details.
```

Comment:
codesigning or code signing?

Reply:
good catch, it seems like it's two words everywhere. I'll make that fix. :+1: